servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/BiLongConsumer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.concurrent.api;
+
+import javax.annotation.Nullable;
+
+/**
+ * A special consumer that takes an {@code long} and a custom arguments.
+ *
+ * @param <T> The other argument to this function.
+ */
+@FunctionalInterface
+public interface BiLongConsumer<T> {
```

Comment:
This is very similar to `java.util.function.ObjLongConsumer`. The argument order is just flipped.

Reply:
good catch! I will use this instead.